applications/MeshingApplication/CMakeLists.txt

```
@@ -58,22 +58,26 @@ endif(${KRATOS_BUILD_TESTING} MATCHES ON)
 
 add_library(KratosMeshingCore SHARED ${KRATOS_MESHING_APPLICATION_CORE} ${KRATOS_MESHING_APPLICATION_CORE_MMG} ${KRATOS_MESHING_APPLICATION_TESTING_SOURCES})
 
+IF(NOT DEFINED MMG_LIBRARY_PREFIX)
+    SET(MMG_LIBRARY_PREFIX "lib")
+ENDIF(NOT DEFINED MMG_LIBRARY_PREFIX)
```

can you try this please @marcnunezc?
```suggestion
OPTION ( MMG_LIBRARY_PREFIX "Prefix for MMG lib" "lib")
```

---

Ìf I dont set the MMG_LIBRARY_PREFIX in our configure, this returns:
MMG_LIBRARY_PREFIX="OFF"
which tries to look for the library name
"lib/OFFmmg.a"

Is this the expected behaviour?

---

What I thought of doesn't with string options, please neglect my comment, sorry for the noise and thx for testing :)

---

If it worked it would have been much cleaner!

---

works only with ON/OFF unfortunately :/